packages/salesforcedx-vscode-apex/src/commands/forceApexExecute.ts

```diff
@@ -75,7 +74,7 @@ export class AnonApexGatherer
   }
 }
 
-export class ApexLibraryExecuteExecutor extends LibraryCommandletExecutor<
+export class AnonApexLibraryExecuteExecutor extends LibraryCommandletExecutor<
```

Comment:
Since this is specifically for Anon Apex, I felt adding "Anon" here made the class more clear.

Reply:
We should rename the file as well?
Took a look at the file....we have multiple exported classes from the same file...uh that seems unusual.

Reply:
Done